partners/balancer.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Balancer",
+  "description": "Balancer is a decentralized automated market maker (AMM) protocol that represents a flexible building block for programmable liquidity.",
+  "imgUrl": "https://raw.githubusercontent.com/balancer/brand-assets/main/logo/circle-container/logo-balancer-black-128x128.svg",
+  "missions": [
+    {
```

Comment:
Only one mission is allowed per partner, you can have multiple tasks under the quests